### ISSUE TITLE
Target Product Profile (Activity)

### ISSUE THREAD

**Issue by danaklug** [opensourceantibiotics/Series-2-Diarylimidazoles#9]
@rhanson1046 has proposed the following guidelines for a target product profile of a lead compound. It would be good to come to a consensus with all contributors as to whether these are reasonable targets, and if there are other things we should be looking for in a lead compound. This issue is a space for that discussion.

- Target organism MIC: <0.5 µg/mL

- Host cell MIC: >64 µg/mL

- Selectivity vs other cell types (i.e., antifungal/antiviral)

- Low resistance potential

**Comment by drc007** [comment]
@danaklug Good idea, I always feel that items in a target product profile (TPP) should be quantifiable, and well defined.

So what is the target organism? what cell type for Host cell MIC, what degree of selectivity and what is meant by antiviral cell type, how would you quantify resistance potential?

I'd add, what is the intended clinical indication/route of administration, this would impact the desired pharmacokinetics.

**Comment by danaklug** [comment]
@drc007 The small array synthesized at UNC is active against MRSA and has tox data against HEK293 cells. It would be good in future to test against the same cell line if possible but as of now I don't think we have any tox assays lined up (see [#3](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/issues/3) ).

IMO, oral administration is ideal but as it's an antibiotic we have a bit of flexibility there. See [#10](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/issues/10) for focused ADME/PK discussion.

As for the rest, nothing is set in stone and these things are open for further discussion!
